pkg/apigee/apigee.go

```diff
@@ -189,13 +191,165 @@ func (c *Client) hostAliases(org, env, virtualHost string) ([]string, error) {
 	return hosts, nil
 }
 
+// CPSEnabledForOrg returns a bool indicating whether the requested Org has CPS enabled or not
+func (c *Client) CPSEnabledForOrg(orgName string) (bool, error) {
+	c.initDefaults()
+
+	cpsEnabled := false
+
+	orgURL := fmt.Sprintf("%sv1/organizations/%s", c.ApigeeAPIHost, orgName)
+	resp, err := c.Get(orgURL)
+	if err != nil {
+		return false, err
+	}
+
+	defer resp.Body.Close()
+
+	var rawOrg interface{}
+
+	err = json.NewDecoder(resp.Body).Decode(&rawOrg)
+
+	if err != nil {
+		fmt.Printf("Error unmarshalling response: %v\n", err)
+		return false, err
+	}
+
+	org := rawOrg.(map[string]interface{})
+	orgProps := org["properties"].(map[string]interface{})
+	orgProp := orgProps["property"].([]interface{})
+
+	for _, rawProp := range orgProp {
+		prop := rawProp.(map[string]interface{})
+		if prop["name"] == "features.isCpsEnabled" {
+			if prop["value"] == "true" {
+				return true, nil
+			}
+			break
+		}
+	}
+	return cpsEnabled, nil
+}
+
+func (c *Client) CreateKVM(orgName, envName, publicKey string) error {
+	const apigeeKVMName = "shipyard-routing"
```

Comment:
Should we be putting this into the _"shipyard"_ KVM?  If we're already using _"shipyard"_ for the application/environment configuration, I could see us using that KVM for storing the routing key.  _(This is not a request to change, just something to think about.)_